factorio-map-process.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+## Defaults
+SERVER_BASE_PATH=/srv/factorio/maps
+MAP_TILES_PATH=images
+FACTORIO_MAP_SCRIPT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/factoriomap.py
+TARFILE=
+REDUCE=0
+
+function usage() {
+    cat <<-EOF
+USAGE
+
+factorio-map-process.sh [OPTIONS] tarfile
+
+DESCRIPTION
+
+This script will process a given tarfile into a set of images used in a leaflet map.
+The default behavior will create a directory for the world maps if one does not exist and copy the example html page in as 'index.html'
+The tar file will be parsed for a map name and a date, separated by underscores (_). Files should be of the format {WorldName}_YYYY_MM_DD.tar.
```

Comment:
Update description to reflect expecting date in the standard format.

```suggestion
The tar file will be parsed for a map name and a date, separated by underscores (_). Files should be of the format {WorldName}_YYYY-MM-DD.tar.
```